.github/workflows/hello_world_multiplatform.yaml

```diff
@@ -63,6 +63,7 @@ jobs:
         with:
           app-path: zephyr
           toolchains: aarch64-zephyr-elf:arc-zephyr-elf:arc64-zephyr-elf:arm-zephyr-eabi:mips-zephyr-elf:riscv64-zephyr-elf:sparc-zephyr-elf:x86_64-zephyr-elf:xtensa-dc233c_zephyr-elf:xtensa-sample_controller32_zephyr-elf:rx-zephyr-elf
+        ccache-cache-key: hw-${{ os.matrix }}
```

Comment:
```suggestion
          ccache-cache-key: hw-${{ os.matrix }}
```

Reply:
yeah got it, was waiting for the doc build run to run and see if it had any hit, it didn't, set that one to no cache as well